includes/classes/Indexable/Post/SyncManager.php

```diff
@@ -138,6 +138,17 @@ public function action_edited_term( $term_id, $tt_id, $taxonomy ) {
 			return;
 		}
 
+		/**
+		 *  Filter to allow skipping this action in case of custom handling
+		 *
+		 *  @hook ep_skip_action_edited_term
+		 *  @param {bool} $skip Current value of whether to skip running action_edited_term or not
+		 *  @return {bool}  New value of whether to skip running action_edited_term or not
+		 */
+		if ( apply_filters( 'ep_skip_action_edited_term', false ) ) {
```

Comment:
It might make sense to pass the arguments to the filter to allow for better control of the behavior.
```suggestion
		if ( apply_filters( 'ep_skip_action_edited_term', false, $term_id, $tt_id, $taxonomy ) ) {
```